bokeh/command/subcommands/json.py

```diff
@@ -0,0 +1,49 @@
+'''
+
+'''
+from __future__ import print_function
+
+import json
+
+from ..subcommand import Subcommand
+from ..util import build_applications
+
+class JSON(Subcommand):
+    ''' Subcommand to output applications as serialized JSON
+
+    '''
+
+    name = "json"
+
+    help = "Emit serialized JSON for one application"
+
+    def __init__(self, **kwargs):
+        super(JSON, self).__init__(**kwargs)
+
+        self.parser.add_argument(
+            'file',
+            metavar='DIRECTORY-OR-SCRIPT',
+            help="The app directory or script to generate JSON for",
+            default=None
+        )
+
+        self.parser.add_argument(
+            '--indent',
+            metavar='LEVEL',
+            type=int,
+            help="indentation to use when printing",
+            default=None
+        )
+
+        self.parser.add_argument(
+            '--sort',
+            action='store_true',
+            help="Sort JSON keys"
```

Comment:
doc.to_json already sorts, so omitting `--sort` won't do anything right now. But sorted seems like the right default to me anyway (unsorted is a nondeterministic order, which doesn't seem useful). Why would you turn off sorting? If you do want to sometimes, maybe `--no-sort` would be better so the default is sorted.